### PR TITLE
🌐 Add Ukrainian translation for `docs/uk/docs/tutorial/query-params.md`

### PR DESCRIPTION
**Pull request by ss-o-furda**
related: https://github.com/tiangolo/fastapi/issues/1748

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 317c1d6f1ecff747c9738de5f77adf450c1252be at: https://649a13ae05f6af1016d6cef2--fastapi.netlify.app

File: docs/uk/docs/tutorial/query-params.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+```
+http://127.0.0.1:8000/items/?skip=0&limit=10
+```
+
+...параметрами запиту є:
+
+* `skip`: зі значенням `0`
+* `limit`: зі значенням `10`
+
+Оскільки вони є частиною URL-адреси, вони є «природними» рядками.
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/query-params.md:22]:
```suggestion
Оскільки вони є частиною URL-адреси, вони є «природно» рядками.
```

**Review by romashevchenko — CHANGES_REQUESTED** [review]
Hi there!
It's Romana from DreamProIT. I've reviewed your translation and added a few suggestions.

**Comment by rostik1410** [comment]
@ss-o-furda @romashevchenko  Hey, will anyone finish this?) I'm ready to approve, let's add more Ukraine translates. Stay safe, hope you guys are okay.
P.S. If nobody wants to finish could I create my own PR  based on this one with addressed comments?

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**